docs/visualize/timelion.asciidoc

```diff
@@ -504,3 +504,61 @@ image::images/timelion-conditional04.png[]
 {nbsp}
 
 For additional information on Timelion conditional capabilities, go to https://www.elastic.co/blog/timeseries-if-then-else-with-timelion[I have but one .condition()].
+
+[float]
+[[timelion-deprecation]]
+=== Timelion App deprecation
+
+In *8.0* and later, the Timelion app is going to be deprecated. There are actions you need to complete to copy your existing Timelion worksheets to a dashboard.  
+
+NOTE: Only the Timelion app is deprecated, not the Timelion editor. 
+
+[role="screenshot"]
+image::images/timelion-app.png[]
+{nbsp}
+
+[float]
+[[timelion-app-to-vis]]
+==== Create Timelion visualizations from a Timelion worksheet
```

Comment:
I think we should already focus on the "dashboard centric" approach here, and say first how to create a new dashboard and from that dashboard use the "Create new" button to create the timlion visusalizations. Otherwise we're kind of fighting our own dashboard documentation trying to get people to move to a dashboard-first workflow.